azurelinuxagent/ga/update.py

```diff
@@ -464,19 +535,22 @@ def _emit_restart_event(self):
     def _emit_changes_in_default_configuration():
         try:
             def log_event(msg):
-                logger.info(msg)
+                logger.info("******** {0} ********", msg)
```

Comment:
we had an incident where the GoalStatePeriod was changed, but we did not notice the log statement... I tried to do this message more visible by adding asterisks here and an explanatory note a few lines below